hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/ContainerStateMachine.java

```diff
@@ -258,8 +257,9 @@ private long loadSnapshot(SingleFileSnapshotInfo snapshot)
    * @throws IOException
    */
   public void persistContainerSet(OutputStream out) throws IOException {
```

Comment:
should we rename this function as well?
given that other instances of containerSet have been renamed.

Reply:
will address it the next patch.